packages/core/src/components/button/buttons.tsx

```diff
@@ -84,9 +131,10 @@ export class AnchorButton extends React.Component<React.HTMLProps<HTMLAnchorElem
 
 export const AnchorButtonFactory = React.createFactory(AnchorButton);
 
-function getButtonClasses(props: IButtonProps) {
+function getButtonClasses(props: IButtonProps, isActive: boolean = false) {
```

Comment:
minor: I don't think you need the typedef if you provide a default parameter value